test/runners/julia_spec.js

```diff
@@ -23,7 +23,7 @@ describe('julia runner', function() {
         `,
         testFramework: 'Test'
       }, function(buffer) {
-        expect(buffer.stdout).to.equal('("<PASSED::>Test Passed","<:LF:>\\n")');
+        expect(buffer.stdout).to.equal('("<PASSED::>Test Passed", "<:LF:>\\n")');
```

Comment:
`<PASSED::>Test Passed`
You can see the [documentation](https://github.com/Codewars/codewars-runner-cli/blob/master/documentation/output_format.md) for other cases.